Storage/ThirdParty/SwiftData.swift

```diff
@@ -406,7 +509,11 @@ open class ConcreteSQLiteDBConnection: SQLiteDBConnection {
         return nil
     }
 
-    func interrupt() {
+    public func setVersion(_ version: Int) -> NSError? {
```

Comment:
Why does this need to be public? Surely it's only used by `prepareSchema`?

Reply:
This is also used internally in `BrowserSchema` for getting through the intermediate migrations prior to `PRAGMA user_version`.